ibm/service/power/resource_ibm_pi_volume.go

```diff
@@ -38,156 +37,156 @@ func ResourceIBMPIVolume() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			helpers.PICloudInstanceId: {
-				Type:        schema.TypeString,
-				Required:    true,
+			Arg_CloudInstanceID: {
 				Description: "Cloud Instance ID - This is the service_instance_id.",
-			},
-			helpers.PIVolumeName: {
-				Type:        schema.TypeString,
 				Required:    true,
+				Type:        schema.TypeString,
+			},
+			Arg_VolumeName: {
 				Description: "Volume Name to create",
+				Required:    true,
+				Type:        schema.TypeString,
 			},
-			helpers.PIVolumeShareable: {
-				Type:        schema.TypeBool,
-				Optional:    true,
+			Arg_VolumeShareable: {
 				Description: "Flag to indicate if the volume can be shared across multiple instances.",
+				Optional:    true,
+				Type:        schema.TypeBool,
 			},
-			helpers.PIVolumeSize: {
-				Type:        schema.TypeFloat,
-				Required:    true,
+			Arg_VolumeSize: {
 				Description: "Size of the volume in GB",
+				Required:    true,
+				Type:        schema.TypeFloat,
 			},
-			helpers.PIVolumeType: {
-				Type:             schema.TypeString,
-				Optional:         true,
+			Arg_VolumeType: {
 				Computed:         true,
-				ValidateFunc:     validate.ValidateAllowedStringValues([]string{"tier0", "tier1", "tier3", "tier5k"}),
 				DiffSuppressFunc: flex.ApplyOnce,
 				Description:      "Type of disk, if disk type is not provided the disk type will default to tier3",
-			},
-			helpers.PIVolumePool: {
-				Type:             schema.TypeString,
 				Optional:         true,
+				ValidateFunc:     validate.ValidateAllowedStringValues([]string{"tier0", "tier1", "tier3", "tier5k"}),
+				Type:             schema.TypeString,
+			},
+			Arg_VolumePool: {
 				Computed:         true,
 				DiffSuppressFunc: flex.ApplyOnce,
 				Description:      "Volume pool where the volume will be created; if provided then pi_affinity_policy values will be ignored",
+				Optional:         true,
+				Type:             schema.TypeString,
 			},
 			PIAffinityPolicy: {
```

Comment:
This should be rename to `Arg_AffinityPolicy`

Reply:
added